### ISSUE TITLE
npm build

### ISSUE THREAD

**Issue by rafael-custodio** [iam4x/pokemongo-webspoof#428]
I keep getting the following error

```
WARNING in DedupePlugin: This plugin was removed from webpack. remove it from configuration.

ERROR in multi main
Module not found: Error: Can't resolve 'babel' in '/Users/rafaelcustodio/contrib/pokemongo-webspoof'
BREAKING CHANGE: It's no longer allowed to omit the '-loader' prefix when using loaders.
                 You need to specify 'babel-loader' instead of 'babel'.
 @ multi main
```

**Comment by eddyhan** [comment]
+1 same problem.

**Comment by iam4x** [comment]
will fix 👍

**Comment by larkinwhitaker** [comment]
Resolved in pull request #431
